src/Shared/SignalR/FunctionalTestBase.cs

```diff
@@ -36,9 +36,9 @@ public FunctionalTestBase()
             };
         }
 
-        public Task<InProcessTestServer<T>> StartServer<T>(Func<WriteContext, bool> expectedErrorsFilter = null) where T : class
+        public Task<InProcessTestServer<T>> StartServer<T>(Func<WriteContext, bool> expectedErrorsFilter = null, bool verifyLogs = true) where T : class
```

Comment:
Can the tests that use this instead pass in a func that just returns true?

Reply:
Ah -- yep! That would work.